src/commands/deployWorkspaceProject/getDefaultValues/getWorkspaceProjectPaths.ts

```diff
@@ -14,6 +14,8 @@ export async function getWorkspaceProjectPaths(context: IActionContext): Promise
     const rootFolder: WorkspaceFolder | undefined = await getRootWorkspaceFolder(prompt);
 
```

Comment:
Mentioned it somewhere else, but I'd mark `context.telemetry.properties.hasWorkspaceProjectOpen = 'true';` here so that it's easier to compare against. I think it just helps my brain more when I see it in the dashboard.